mst/saml-okta-mst.md

```diff
@@ -0,0 +1,134 @@
+    `Method Type` as `SAML`.
+
+1.  Choose the team to add invited people or leave it blank and click `Add`.
+
+1.  The two parameter `Metadata URL`, and `ACS URL` to setup the SAML
+    authentication in your Identity Provider appears.
+
+</procedure>
+
+## Creating the Okta application and assigning users
+
+This is a two step process. First create the SAML SP-Initiated
+authentication flow, then create a bookmark app that redirects to
+the Managed Service for TimescaleDB login page.
+
+<procedure>
+
+1.  Login to the `Admin` portal and navigate to the `Applications` tab.
+
+1.  Click `Add Application` and click `Create New App` You should see
+    the `Create a new Application Integration` form.
+
+1.  Select `SAML 2.0` for the `Sign on method`, then click `Create`.
+
+1.  In the `Create SAML Integration` set the `App name`, `App logo`, and `App visibility`,
+    then click `Next`.
+
+1.  Update the `Configure SAML` page with these values:
+    |Parameter|Example value|Description|
+   |-|-|-|
+   |Single sign on URL|`https://api.timescale.io/v1/sso/saml/account/{account_id}/method/{account_authentication_method_id/acs`|
+   The value in Timescale Console on the newly created `Authentication method` page.|
+   |Audience URI (SP Entity ID)|`https://api.timescale.io/v1/sso/saml/account/{account_id}/method/{account_authentication_method_id}/metadata`|
+   The value in Timescale Console on the newly created `Authentication method` page.|
+   |Default Relay State|https://portal.managed.timescale.com/| is the homepage
+   of the MST Console and is fundamental for IdP initiated sign-on to function
+   correctly.|
+
+1.  In the `ATTRIBUTE STATEMENTS`should have an entry with these values:
+
+   |Parameter|Example value|
+   |-|-|
+   |name|`email`|
+   |value|`user.email`|
+
+1.  Click `Next` then `Finish`. You will be redirect to your application in Okta.
+
+1.  In the `Assignments` tab of the application on Okta, click the `Assign` to assign
+    users or groups to the application.
+
+1.  In the `Sign On` tab of the application on Okta, click the `View Setup Instructions`
+    and gather the required information to configure Okta in MST.
+1.  Download the *Certificate file* that you require to configure Okta in MST.
+
+</procedure>
+
+## Set up the Okta Application in Managed Service for TimescaleDB
+
+<procedure>
+
+1.  Login to the [MST account] [mst-login].
+1.  Finalize the configuration on the `Authentication` tab of the `Account page`.
+1.  Add the *Certificate file* in the `SAML Certificate` field.
+1.  Enable the `Enable IdP Login` and `Enable authentication method`.
+1.  Click `Edit methd` to save the settings.
+
+</procedure>
+
+You can now use the `Account Link URL` on the authentication configuration page
+to link your Okta account and MST profile. You can also invite other members of
+your team to login or signup to MST using Okta with the **Signup link** shown in
+the `Authentication method` page.
+
+## Troubleshooting
```

Comment:
We should put these into the new Troubleshooting format. @charislam can help if you need it. 

Also, make sure you're writing these in CCFR format.

Reply:
added .md files to the TS directory